style.css

```diff
@@ -205,25 +204,29 @@ aside{
 
 select{
 	height: 40px;
-	width: 390px;
-	background: #e4e4e4;
+	width: 370px;
+	border: none;
+	position: relative;
+	background: url(custom-arrow.png) 96% 15% no-repeat #e4e4e4;
+	-webkit-appearance: none;
+  -moz-appearance: none;
```

Comment:
In my text editor it displays properly. Don't know why there is additional indentation once I push to GH.